common/common-utils/src/main/java/com/duckduckgo/common/utils/extensions/StringExtensions.kt

```diff
@@ -87,3 +87,5 @@ fun String.compareSemanticVersion(targetVersion: String): Int? {
     }
     return 0
 }
+
+fun Boolean.toBinaryString(): String = if (this) "1" else "0"
```

Comment:
🔍 I think this is probably more appropriate in a `BooleanExtensions` file

Reply:
You’re right.. Fixed!